packages/market-service/src/foxy/foxy.test.ts

```diff
@@ -40,7 +43,7 @@ describe('foxy market service', () => {
     })
 
     it('can handle rate limiting', async () => {
-      mockedAxios.get.mockResolvedValue({ status: 429 })
+      mockedAxios.get.mockRejectedValue({ status: 429 })
```

Comment:
Note: Since the underlying `CoinGeckoMarketService` has its own `RATE_LIMIT_THRESHOLDS_PER_MINUTE.COINGECKO` (500) rate limiting, we might want to ditch this test altogether, but might as well keep it for safety